tools/libboard/board_d5005/board_d5005.c

```diff
@@ -127,7 +127,7 @@ fpga_result read_max10fw_version(fpga_token token, char *max10fw_ver, size_t len
 		return FPGA_INVALID_PARAM;
 	}
 
-	res = read_sysfs(token, DFL_SYSFS_MAX10_VER, buf, FPGA_VAR_BUF_LEN);
+	res = read_sysfs(token, DFL_SYSFS_MAX10_VER, buf, FPGA_VAR_BUF_LEN -1);
```

Comment:
Could we make the spacing around the -1 consistent?  eg, line 160 uses
```c
MACADDR_LEN-1
```
Let's converge on the form that uses a space on either side of the -
```c
X - 1
```